search/includes/classes/class-queuewaittimejob.php

```diff
@@ -2,14 +2,12 @@
 
 namespace Automattic\VIP\Search;
 
-class QueueWaitTimeJob {
-	const CRON_EVENT_NAME = 'vip_config_sync_cron';
-
-	const FIELD_COUNT_GAUGE_DISABLED_SITES = array();
```

Comment:
Removed `FIELD_COUNT_GAUGE_DISABLED_SITES` since it doesn't seem to be used anywhere.